var/spack/repos/builtin/packages/scorep/package.py

```diff
@@ -90,7 +90,7 @@ def url_for_version(self, version):
     # two components of cube -- cubew and cubelib.
 
     # SCOREP 8
-    depends_on("binutils", type="link", when="@8:")
+    depends_on("binutils libs=shared", type="link", when="@8:")
```

Comment:
After reading #37004 I'm not entirely sure whether you want to disable static binutils libraries or whether it is enough to have shared ones available. Will the `--disable-static` below ignore static binutils libraries? If so, this should work, if not: At least for me, this still concretizes a binutils with `libs=shared,static`. I can force it with `scorep ^binutils libs=shared`, however. Adding `conflicts("binutils libs=static")` also makes it concretize to `libs=shared`.

Reply:
While this `conflicts` clause would be my preferred solution, it appears to be failing `package-audits` in CI, claiming that binutils has no `libs` variant, which seems to be total nonsense to me. @michaelkuhn any hints here?